themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
```

Comment:
I think it makes sense to fix at the same time; it's just that there's a lot of changes here for me to review properly.